pkgs/games/leela-zero/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, fetchFromGitHub, cmake, boost, eigen
+, opencl-headers, ocl-icd, qtbase , zlib }:
+
+stdenv.mkDerivation rec {
+  name = "leela-zero-${version}";
+  version = "0.16";
+
+  src = fetchFromGitHub {
+    owner = "gcp";
+    repo = "leela-zero";
+    rev = "4fd6e694ead45b7cbb14ea76c5bdfcbfa662304b";
```

Pinning to tag is more readable:
```suggestion
    rev = "v${version}";
```

---

Gah. I saw too late I could have just taken the suggested changes.